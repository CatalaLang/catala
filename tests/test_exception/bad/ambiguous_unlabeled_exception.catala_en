## Test

```catala
declaration scope A:
  context x content integer

scope A:
  definition x equals 0

  definition x equals 1

  exception
  definition x equals 2
```
