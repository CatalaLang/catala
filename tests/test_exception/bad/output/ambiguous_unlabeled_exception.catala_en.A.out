[ERROR] This exception can refer to several definitions. Try using labels to disambiguate
[ERROR] 
[ERROR] Ambiguous exception
[ERROR]   --> test_exception/bad/ambiguous_unlabeled_exception.catala_en
[ERROR]    | 
[ERROR] 10 |   definition x equals 1
[ERROR]    |                       
[ERROR] 11 | 
[ERROR]    | 
[ERROR] 12 |   exception
[ERROR]    |   ^^^^^^^^^
[ERROR] 13 |   definition x equals 2
[ERROR]    |   ^^^^^^^^^^^^^^^^^^^^^^
[ERROR]    + Test
[ERROR] 
[ERROR] Candidate definition
[ERROR]   --> test_exception/bad/ambiguous_unlabeled_exception.catala_en
[ERROR]    | 
[ERROR] 10 |   definition x equals 1
[ERROR]    |              ^
[ERROR]    + Test
[ERROR] 
[ERROR] Candidate definition
[ERROR]  --> test_exception/bad/ambiguous_unlabeled_exception.catala_en
[ERROR]   | 
[ERROR] 8 |   definition x equals 0
[ERROR]   |              ^
[ERROR]   + Test
